spec/unit/puppet_x/chocolatey/chocolatey_common_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'spec_helper'
+require 'puppet_x/chocolatey/chocolatey_install'
+
+describe 'Chocolatey Common' do
+
+  let (:first_compiled_choco_version) {'0.9.9.0'}
+  let (:newer_choco_version) {'0.9.10.0'}
+  let (:last_posh_choco_version) {'0.9.8.33'}
+
+  context ".choco_version" do
+    it "should return PuppetX::Chocolatey::ChocolateyVersion.version" do
+      expected = '0.9.9.0.1'
```

Comment:
Yes, it's so that the verification is valid. I could accidentally have a false positive on a version if the version was a real one.